### DISCUSSION TITLE
it even get started!

### DISCUSSION THREAD

**Discussion by alansenairj** [hlky/stable-diffusion#287] [Q&A]
./linux-sd.sh: line 8: syntax error near unexpected token `newline'
./linux-sd.sh: line 8: `<!DOCTYPE html>'

???

Fedora 35
RTX 2060 8 GB

**Comment by rfwatkins** [comment]
Having a similar issue with linux-sd.sh, syntax error on line 34, expecting to find "fi", but I can't see any obvious problem there.
